pages/documentation/entropy/protocol-design.mdx

```diff
@@ -3,47 +3,47 @@
 The Entropy protocol is an extension of a classical commit/reveal protocol.
 The original version has the following steps:
 
-1. Two parties A and B each draw secret random numbers, $$x_A$$ and $$x_B$$.
-2. A and B hash their random numbers and share the hashes, $$h_{A} = hash(x_{A})$$ and $$h_{B} = hash(x_{B})$$
-3. A and B reveal $$x_{A}$$ and $$x_{B}$$
-4. Both parties verify that $$hash(x_{A}) == h_{A}$$ and $$hash(x_{B}) == h_{B}$$
-5. The random number $$r = hash(x_A, x_B)$$
+1. Two parties A and B each draw secret random numbers, `x_A` and `x_B`.
```

Comment:
Isn't the usage of {} a bit inconsistent? Is there a reason behind it?

Reply:
ah... both of these used to render the same in latex, but they look different now. Fixed.